seed/challenges/03-front-end-libraries/react-and-redux.json

```diff
@@ -50,7 +50,7 @@
       },
       "tests": [
         "assert((function() { const mockedComponent = Enzyme.mount(React.createElement(DisplayMessages)); return mockedComponent.find('div').text() === '' })(), 'message: The <code>DisplayMessages</code> component should render an empty <code>div</code> element.');",
-        "getUserInput => assert((function() { const noWhiteSpace = getUserInput('index').replace(/ /g,''); return noWhiteSpace.includes('constructor(props)') && noWhiteSpace.includes('super(props'); })(), 'message: The <code>DisplayMessages</code> constructor should be called properly with <code>super</code>, passing in <code>props</code>.');",
+        "assert((function() { const noWhiteSpace = code.replace(/\\s/g,''); return noWhiteSpace.includes('constructor(props)') && noWhiteSpace.includes('super(props'); })(), 'message: The <code>DisplayMessages</code> constructor should be called properly with <code>super</code>, passing in <code>props</code>.');",
```

Comment:
`code` is legacy. We use getUserInput as interface to get the raw user input.

Reply:
Ok, we can revert this then. We should also mention this on #16201

Reply:
@no-stack-dub-sack did you get a chance to revert this? Is there anything else necessary before we can merge this?

Reply:
I'm about to push the changes! Was offline all day yesterday or would have gotten to it sooner. I think other than this, a final run through by someone independent, running through all challenges and solving them would be a great last step.